src/gradualizer.erl

```diff
@@ -196,3 +196,13 @@ options_from_forms([{attribute, _L, gradualizer, Opt} | Fs]) ->
     [Opt | options_from_forms(Fs)];
 options_from_forms([_F | Fs]) -> options_from_forms(Fs);
 options_from_forms([]) -> [].
+
+%% Normalize options so that module local options override options
+%% from the top level
+normalize_options(ModOpts, Opts) ->
+    proplists:normalize(ModOpts ++ Opts ++ [prelude] % using prelude is the default
+		       ,[{negations, [{no_infer, infer}
+				     ,{no_prelude, prelude}
```

Comment:
`--no-infer` is returned as `{infer, false}` from CLI already.

Reply:
Good point.